File: android-client/data/src/main/java/ru/hits/hubabank/data/network/bill/model/BillDto.kt
package ru.hits.hubabank.data.network.bill.model

import kotlinx.serialization.Serializable
import ru.hits.hubabank.domain.bill.model.Bill

@Serializable
internal data class BillDto(
    val id: String,
    val userId: String,
    val balance: Long,
    val isHidden: Boolean,
)

internal fun BillDto.toDomain(): Bill {
    return Bill(
        id = id,
        balance = balance,
        isHidden = isHidden,
    )
}

File: BFF-client/BFF-client.Api/Models/bill/BillDto.cs
namespace BFF_client.Api.model.bill
{
    public class BillDto
    {
        public string Id { get; set; }

        public string UserId { get; set; }

        public long Balance { get; set; }
        
        public string Currency {  get; set; }

        public bool isHidden { get; set; }
    }
}
